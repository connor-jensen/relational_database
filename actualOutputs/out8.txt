Schedule('M',C,'236')? Yes(1)
  C='236'