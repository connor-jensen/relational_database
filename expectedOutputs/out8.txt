Schedule('M',C)? Yes(2)
  C='113'
  C='236'